src/NHibernate/Loader/JoinWalker.cs

```diff
@@ -439,31 +461,46 @@ internal OuterJoinableAssociation InitAssociation(OuterJoinableAssociation assoc
 		/// For an entity class, add to a list of associations to be fetched
 		/// by outerjoin
 		/// </summary>
-		protected virtual void WalkEntityTree(IOuterJoinLoadable persister, string alias, string path, int currentDepth)
+		/// Since 5.4
+		protected virtual void WalkEntityTree(IOuterJoinLoadable persister, string alias, string path)
```

Comment:
Is this a new method?

Reply:
New overload without `currentDepth`. Old one is `Obsolete` now.. `Since...` comment need to be moved to old one...